dev/cnf/resources/bnd/rejar.props

```diff
@@ -32,7 +32,7 @@ Bundle-Version: ${bFullVersion}.${if;${driver;gradle};${version.qualifier};eclip
 -consumer-policy :    ${range;[==,+)}
 
 # Require a 1.6 JDK (at least): see section 3.4.1 of OSGi 4.3 core spec
-Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
```

Comment:
Please update the comment.

Reply:
The comment is accurate in that Java 6 is the minimum required level.  If you don't want it left, I am included to remove it completely instead of updating it.

Reply:
I have removed the comment.

Reply:
Yeah OK, removed is fine